.github/workflows/release.yml

```diff
@@ -16,44 +17,38 @@ jobs:
         with:
           token: ${{ secrets.GH_TOKEN }}
 
-      - name: Use Node.js 20.17.0
+      - name: Set git user to carbon-bot
+        run: |
+          git config --global user.email "carbon@us.ibm.com"
+          git config --global user.name "carbon-bot"
+
+      - name: Use Node.js 20.x LTS
         uses: actions/setup-node@v4
         with:
-          node-version: '20.17.0'
+          node-version: '20.x'
+          cache: 'yarn'
+          registry-url: 'https://registry.npmjs.org/'
+          always-auth: true
+          auth-token: ${{ secrets.NPM_TOKEN }}
```

Comment:
Eliminate need for explicit minor versions, set yarn cache and npmjs authentication here instead of below.